tests/common/plugins/sanity_check/recover.py

```diff
@@ -26,13 +27,47 @@ def reboot_dut(dut, localhost, cmd):
         assert False, "Failed to reboot the DUT"
 
     localhost.wait_for(host=dut.hostname, port=22, state="started", delay=10, timeout=300)
-    wait(30, msg="Wait 30 seconds for system to be stable.")
+    wait(120, msg="Wait 120 seconds for system to be stable.")
 
 
-def recover(dut, localhost, recover_method):
+def __recover_interfaces(dut, fanouthosts, result):
+    for port in result['down_ports']:
+        logging.debug("Restoring port {}".format(port))
+        fanout, fanout_port = fanout_switch_port_lookup(fanouthosts, port)
+        if fanout and fanout_port`:
```

Comment:
I saw it in the prev diff and while commenting, I got a message that it is outdate. I believe there an extra '`'.

Reply:
Yeah, I fixed it. Thanks :-)